docs/webserver.md

```diff
@@ -0,0 +1,41 @@
+# Webserver
+
+| | web02.rackspace.theforeman.org |
+| - | - |
+| type | OpenStack VM |
+| OS | CentOS 7 |
+| CPUs | 2 |
+| RAM | 2GB |
+| Storage | /dev/xvda (40GB): root, /dev/xvdb (100GB) + /dev/xvdc (50GB): data |
+| Managed by | [web.pp](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/profiles/manifests/web.pp) |
+
+## Domains
+
+These domains are all hosted on the webserver.
+
+* theforeman.org, www.theforeman.org
+* deb.theforeman.org
+* debugs.theforeman.org
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+* yum.theforeman.org
```

Comment:
these three are fronted by Fastly, and web02 is only the backend, wonder if we can somehow write that down nicely?